tests/data/expected/main/main_use_default_kwarg/output.py

```diff
@@ -72,3 +72,7 @@ class Name(BaseModel):
 
 class Tag(BaseModel):
     __root__: str
+
+
+class Notes(BaseModel):
+    comments: Optional[List[str]] = Field(default_factory=list)
```

Comment:
this should not have an optional.

Reply:
But, if the object has `default_factory`, may remove `Optional[]` for the behavior :thinking:

Reply:
@koxudaxi ok thanks, trying now! I dont understand why you would ever have Optional if theres. adefault facory though

Reply:
@koxudaxi i confirmed this works using that flag, thank you!